src/plugins/intel_cpu/CMakeLists.txt

```diff
@@ -287,14 +287,14 @@ cross_compiled_file(${TARGET_NAME}
         NAMESPACE   ov::Extensions::Cpu::XARCH
 )
 cross_compiled_file(${TARGET_NAME}
-        ARCH AVX512F AVX2 NEON_FP16 ANY
+        ARCH AVX512F AVX2 NEON_FP16 SVE ANY
                     src/nodes/kernels/scaled_attn/softmax.cpp
         API         src/nodes/kernels/scaled_attn/softmax.hpp
         NAME        attn_softmax
         NAMESPACE   ov::Extensions::Cpu::XARCH
 )
 cross_compiled_file(${TARGET_NAME}
-        ARCH AVX512F AVX2 NEON_FP16 ANY
+        ARCH AVX512F AVX2 NEON_FP16 SVE ANY
```

Comment:
looks like you need to add `SVE` only if compiler supports it?

Reply:
Sorry, I couldn't think of how that can be done. Could you please help with the expected change?

Reply:
I think you need to check via internet what compilers (and versions) support SVE and pass `SVE` option to `cross_compiled_file` only for those supported versions / compilers.

Otherwise, we can end up with situation when you are compiling for SVE, but compiler (or its version) does not support it. E.g. first link says SVE are available from gcc 10.x https://stackoverflow.com/questions/56905070/sve-sve2-support-in-gnu-toolchain; the same should be checked for clang and MSVC

Reply:
My latest commit adds compiler checks in both places before adding SVE. A colleague of mine recommended using a small SVE snippet and checking if compilation for that succeeds, then using that as a variable to determine SVE support from the compiler. Please let me know if that works.